lang/python/module/chemfiles.cpp

```diff
@@ -0,0 +1,270 @@
+#include "chemfiles.hpp"
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wmissing-prototypes"
+#pragma clang diagnostic ignored "-Wold-style-cast"
+#pragma clang diagnostic ignored "-Wmissing-braces"
+#pragma clang diagnostic ignored "-Wdocumentation"
+#pragma clang diagnostic ignored "-Wdeprecated"
+#pragma clang diagnostic ignored "-Wsign-conversion"
+
+#pragma GCC diagnostic ignored "-Wold-style-cast"
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl_bind.h>
+namespace py = pybind11;
+
+template<typename T>
+bool check(const chemfiles::optional<T>& o) {
+    return o != chemfiles::nullopt;
+}
+
+template<typename T>
+T get(const chemfiles::optional<T>& o) {
```

Comment:
That's interesting! Why not use the python bindings provided with chemfiles? Is there something specific that you need to do (such as extracting the CHFL_XXX pointers from Python to C++)?

Reply:
Or the ability to use your custom fork of chemfiles (which should work but is not documented at all)?

Reply:
> That's interesting! Why not use the python bindings provided with chemfiles? Is there something specific that you need to do (such as extracting the CHFL_XXX pointers from Python to C++)?

Yes, I would need to do this and ensure that the Python version of Chemfiles is installed! I originally did this in the interest of simplicity, but I am willing to revisit this once the paper is submitted. ACS usually responds within a month, we will know if it goes out for review within a week. The editor will not throughly check GitHub, but will want to see the code is availible.